pkgs/development/libraries/torch/default.nix

```diff
@@ -1,38 +0,0 @@
-{stdenv, fetchgit, luajit, openblas, imagemagick, cmake, curl, fftw, gnuplot
-  , libjpeg, zeromq3, ncurses, openssl, libpng, qt4, readline, unzip
-  , pkgconfig, zlib, libX11, which
-  }:
-stdenv.mkDerivation rec{
-  version = "0.0pre20160820";
-  pname = "torch";
-  buildInputs = [
-    luajit openblas imagemagick cmake curl fftw gnuplot unzip qt4
-    libjpeg zeromq3 ncurses openssl libpng readline pkgconfig
-    zlib libX11 which
-  ];
-
-  src = fetchgit {
-    url = "https://github.com/torch/distro";
```

Comment:
We're running version `0.0pre20160820`, which hasn't built in a long time, and upstream https://github.com/torch/distro/blob/master/README.md says, as of 3 years ago:

> NOTE: Torch is not actively developed anymore and is in maintenance mode.